zenith/src/main.rs

```diff
@@ -319,23 +346,43 @@ fn get_branch_infos(env: &local_env::LocalEnv) -> Result<HashMap<ZTimelineId, Br
     Ok(branch_infos)
 }
 
+fn handle_tenant(tenant_match: &ArgMatches, env: &local_env::LocalEnv) -> Result<()> {
+    let pageserver = PageServerNode::from_env(&env);
+    if let Some(tenantid) = tenant_match.value_of("tenantid") {
+        pageserver.tenant_create(&ZTenantId::from_str(tenantid)?)?
+    } else {
+        for tenant in pageserver.tenants_list()? {
+            println!("{}", tenant);
+        }
```

Comment:
Do we want to expose tenats list in CLI?
If so, let's be consistent with other commands (i.e. `pg list`) and introduce `zenith tenant list`

Reply:
It was the original idea, also it is useful in tests.

`zenith tenant` is actually consistent with `zenith branch`, the logic is equivalent. When there is no argument then display the list and if there is an argument then create the entity. So probably if we want to unify this we need to change `zenith branch` also. Is it suitable here? Or I can change tenant to match pg behaviour and fix branch separately? Which one is preferable?

Reply:
IMO we would change `zenith branch` to smth like `zenith branch list` because we can't really add any subcommands there -- any subcommand would be treated as a branch name. So +1 on sticking with `pg list` syntax.